packages/sitecore-jss-angular/src/components/default-empty-field-editing-image.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+
+/**
+ * Default component that will be rendered in pages when image field is empty.
+ */
+@Component({
+  selector: 'app-default-empty-field-editing-image',
+  template: `
+    <img
+      alt=""
```

Comment:
Should we a bit more descriptive text here?
`Placeholder image for empty field`

Reply:
i renamed the default placeholders, keeping the names similar to what we have in react for consistency